docs/src/remap/remap.adoc

```diff
@@ -301,7 +301,7 @@ Note that while many combinations of argspec options are possible, not all of th
 
 `argspec=`__<words>__ `ngc=`__<procname>__ `modalgroup=`___<group>__::
   The recommended way to call an NGC procedure with a standard argspec parameter conversion.
-  Used if argspec is good enough. Note it's not good enough for remapping the `Tx` and `M6`/`M61` tool change codes.
+  Used if argspec is good enough. Note it's not good enough for remapping the `T`__x__ and `M6`/`M61` tool change codes.
```

Comment:
This isn't formatted correctly. But weird that exactly this works in headings.

Reply:
The `Tx` is wrong since "Tx" is not a command.  I would even consider it a grave (a semantic) mistake in the documentation that needs to be fixed. `T`_`x`_ is what I would like to see without that nasty wide separation, but have no idea how to get there.

Reply:
>Tx is what I would like to see without that nasty wide separation, but have no idea how to get there.

AFAIK there is no way using a syntax that is compatible with our current asciidoc version and also with the current version.
So I suggest to abstain from changes like that until we have migrated to the current asciidoctor version.

Reply:
We were discussing the asciidoctor migration somewhere. But yes, I should refrain from anything like that for now.